common4j/src/main/com/microsoft/identity/common/java/flighting/CommonFlight.java

```diff
@@ -103,7 +104,12 @@ public enum CommonFlight implements IFlightConfig {
     /**
      * Flight to make the state parameter required for the switch browser protocol.
      */
-    SWITCH_BROWSER_PROTOCOL_REQUIRES_STATE("SwitchBrowserProtocolRequiresState", false);
+    SWITCH_BROWSER_PROTOCOL_REQUIRES_STATE("SwitchBrowserProtocolRequiresState", false),
+
+    /** Flight to enable the new key generation without PURPOSE_WRAP_KEY. Default is true.
+    * This is applicable for API >= 23
+    */
+    ENABLE_NEW_KEY_GEN_SPEC_FOR_WRAP_WITHOUT_PURPOSE_WRAP_KEY("EnableNewKeyGenSpecForWrapWithoutPurposeWrapKey", true);
```

Comment:
We want this to be true by default?

Reply:
yes. This way we don't use deprecated key spec.

Do you prefer it to be OFF?